typescript/types.ts

```diff
@@ -141,67 +165,23 @@ export type ExecuteResult = {
   execution_count?: number;
 
   /**
-   * A mime-type keyed dictionary of data
+   * The result of executing the prompt.
    */
   data: JSONValue;
 
   /**
-   * Output metadata.
-   */
-  metadata?: {
-    [k: string]: any;
-  };
-};
-
-/**
- * Data displayed as a result of inference.
- * Ignore: this is a work-in-progress
- */
-export type DisplayData = {
-  /**
-   * Type of output.
+   * The MIME type of the result. If not specified, the MIME type will be assumed to be plain text.
    */
-  output_type: "display_data";
-
-  /**
-   * A mime-type keyed dictionary of data
-   */
-  data: {
-    /**
-     * mimetype output (e.g. text/plain), represented as either an array of strings or a string.
-     */
-    [k: string]: string | string[];
-  };
+  mime_type?: string;
```

Comment:
Can you give an example of what this might look like?

Reply:
Discussed offline and went over mime types.